### ISSUE TITLE
Error message missing context

### ISSUE THREAD

**Issue by campoy** [dgraph-io/dgraph#3759]
If you suspect this could be a bug, follow the template.

- What version of Dgraph are you using?

1.0.16

- Have you tried reproducing the issue with latest release?

yes, I could

- What is the hardware spec (RAM, OS)?

N/A

- Steps to reproduce the issue (command/config used to run Dgraph).

Send any invalid request, for instance this one is missing a comma at the end of line 5

```json
{
  "set": [
    {
      "name": "a"
    }
    {
    	"name": "b"
    }
  ]
}
```

- Expected behaviour and actual result.

I would expect an error message telling me about the line or at least context of the syntax error, instead I see "Invalid mutation"

I found this bug while running a mutation of more than 5000 lines, which meant I was completely cluelles until I used jsonlint.

**Comment by martinmr** [comment]
The gql parser already has line and column information but I am not sure about how JSON errors are reported. I'll take a look.

**Comment by campoy** [comment]
Hey @martinmr , could you give an update on this issue?

**Comment by martinmr** [comment]
My bad, this has been fixed for release 1.0 in commit 0ac6942fa7294d49874d67dbb8ae7aca5e834324
